### PR TITLE
data: Agrim-Sharma174

### PR DESCRIPTION
**Pull request by Agrim-Sharma174**
## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4747"></a>

### CODE REVIEW

File: data/Agrim-Sharma174.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+	"name": "Agrim",
+	"type": "personal",
+	"bio": "Smart Contract Developer | Web Developer | Technical Writer",
+	"links": [{
+			"name": "Read my Blogs",
+			"url": "https://agrimsharma.hashnode.dev/",
+			"icon": "FaHashnode"
```
Comment (kumarsonsoff3) [inline data/Agrim-Sharma174.json:8]: You're probably be using an `invalid` icon. It should be `SiHashnode`

```suggestion
			"icon": "SiHashnode"
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by Agrim-Sharma174** [comment]
Review done

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Review by krshkun — APPROVED** [review]
looks great

**Pull request merged**